.husky/pre-commit

```diff
@@ -1 +1,2 @@
-yarn lint-staged
\ No newline at end of file
+yarn lint-staged
+yarn check-format
```

Comment:
no need for this as it will prevent committing file when something completely unrelated is messed up.